src/sections/FAQ/Resources.tsx

```diff
@@ -0,0 +1,41 @@
+import Link from '@components/Link';
+import React from 'react';
+import YouTube from 'react-youtube';
+
+const Resources = () => {
+  return (
+    <div className="flex flex-col justify-center">
+      <div className="lg:grid lg:grid-cols-2 lg:gap-16 max-w-8xl mx-auto">
```

Comment:
haha it's fine. Since mobile needs to be one object on top of another, It also could be a grid with a single grid column (`grid-col-1`). But, we can achieve this by its default property `block`, that's why I'm only applying `display grid` for big screens